packages/flutter_tts/tizen/src/text_to_speech.cc

```diff
@@ -107,8 +137,8 @@ void TextToSpeech::UnregisterCallbacks() {
   tts_unset_error_cb(tts_);
 }
 
-std::vector<std::string> &TextToSpeech::GetSupportedLanaguages() {
-  if (supported_lanaguages_.size() == 0) {
+void TextToSpeech::InitializeSupportedLanaguagesAndVoiceType() {
+  if (supported_lanaguages_.size() == 0 || supported_voice_types_.size() == 0) {
```

Comment:
This function was executed for every API call,
but now it is executed once during the initialization phase.
This condition and lines 163-168 seem unnecessary. what do you think about this?

Reply:
Thank you for your review.
Yes, I will remove the condition.
But, can you explain about "lines 163-168 seem unnecessary" in detail?

Reply:
Before the change, `TextToSpeech::GetSupportedLanaguages()` was called every time `getLanguages` was called.
So it cleared the vector list before adding items to `vector` using the callback of `tts_foreach_supported_voices`.

But now, `TextToSpeech::InitializeSupportedLanaguagesAndVoiceType()` is called only once via `TextToSpeech::Initialize().`
There is no need to clear `supported_lanaguages_` and `supported_voice_types_` as they will not be rewritten.
And `supported_lanaguages_` and `supported_voice_types_` are already empty lists.

I think it will very rare for `supported_language_` to change, although there may be issues with the list synchronization if system specifications change in the middle of the program.

Reply:
I got it. Thanks for your review.